projects/demo/src/modules/components/badge-notification/examples/2/index.ts

```diff
@@ -1,9 +1,13 @@
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiPlatformModule} from '@taiga-ui/cdk';
+import {TuiBadgeNotificationComponent} from '@taiga-ui/kit';
 
 @Component({
+    standalone: true,
     selector: 'tui-badge-notification-example-2',
+    imports: [TuiPlatformModule, TuiBadgeNotificationComponent],
```

Comment:
Is `TuiPlatformModule` actually required for this example ?

Reply:
Yes, we are using tuiPlatform in html template

Reply:
<img width="1019" alt="platform" src="https://github.com/taiga-family/taiga-ui/assets/35179038/51473a38-cc66-4916-879a-3fe33e3925a6">

Reply:
https://github.com/taiga-family/taiga-ui/blob/b36689acb658e3c2ebef88182f90b03b98ef00fc/projects/demo/src/modules/components/badge-notification/examples/2/index.html#L1-L4